mitosheet/src/components/Mito.tsx

```diff
@@ -219,6 +218,54 @@ export const Mito = (props: MitoProps): JSX.Element => {
         previousNumSheetsRef.current = sheetDataArray.length;
     }, [sheetDataArray])
 
+    const previousNumGraphsRef = useRef<number>(Object.keys(analysisData.graphDataJSON).length)
```

Comment:
These effects are _so_ much more followable and consistent compared to how we were previously thinking about this. This is great!